src/app/client/src/app/modules/explore-page/components/explore-page/explore-page.component.ts

```diff
@@ -130,8 +134,21 @@ export class ExplorePageComponent implements OnInit, OnDestroy, AfterViewInit {
                 this.userType = result;
             }
         });
+
+        this.themeSubscription = this.theme$.subscribe((theme) => {
+        });
+        
+        const localDataThemeAttribute = localStorage.getItem('data-mode');
+        if (localDataThemeAttribute) {
+          this.setTheme(localDataThemeAttribute);
+        }
     }
 
+    setTheme(value: string) {
```

Comment:
@mithun30052001  please add the comment description

Reply:
Sure @princegupta1131 i'll update